src/engraving/libmscore/realizedharmony.cpp

```diff
@@ -92,6 +92,9 @@ const RealizedHarmony::PitchMap& RealizedHarmony::notes() const
 const RealizedHarmony::PitchMap RealizedHarmony::generateNotes(int rootTpc, int bassTpc,
                                                                bool literal, Voicing voicing, int transposeOffset) const
 {
+    //cut octaves from offset
+    transposeOffset %= 12;
```

Comment:
Doesn't this need special handling for negative offsets? Those don't work well with the modulo operator (which does _not_ perform the same modulo operation for negative numbers as we know it in mathematics). 
(Or will we never encounter a negative offset here? In that case, let's make that explicit with a comment.)

Also, since we apparently have a named constant for the number of notes per octave (`PITCH_DELTA_OCTAVE`)... let's use it, why not :)

Reply:
Hm, now I doubt about that modulo operator... perhaps it's just fine. I seem to recall that there was _something_ annoying with it, or at least that I've seen some code that tries to work around that... but for this case, the behaviour seems fine to me now.

Reply:
In other languages, it may behave differently, see https://stackoverflow.com/questions/11630321/why-does-c-output-negative-numbers-when-using-modulo
(or may be older implementation too?)

I can do somthing like
```
if (transposeOffset < 0) {
    transposeOffset = -((-transposeOffset) % 12);
} else {
    transposeOffset %= 12;
}
```
but do we need it?

Reply:
No, I think the C++ behaviour is fine for our case! :)